roles/support-engineer.md

```diff
@@ -0,0 +1,30 @@
+Support Engineer
+====================
+
+## The Support Engineer builds and improves Mariana Tek’s user and support experience. 
+* Providing great support to clients is culturally important to Mariana Tek.
+
+### Accountabilities
+
+* Take end-to-end ownership of questions and issues, including initial troubleshooting, root cause analysis and resolution.
+* Push creative thinking beyond the boundaries of the product process to come up with internal process improvements and new ways to delight customers.
+* Collect identify, document, and fix software defects.
+* Respond to realtime incidents affecting the ability of our customers to run their business.
+
+### Responsibilities
+
+#### Support
+
+* Act as a steward of our codebases, guiding product teams in addressing bugs fixes and tech debt.
+
+#### Product Contribution
+
+* Contribute to application codebase in support of quality and reliability
+* Work with Product teams to define and implement short to medium term software changes.
```

Comment:
Let's mention something about working on building out features that reduce the burden on Support.